### PR TITLE
F/styles refactor

### PR DESCRIPTION
**Pull request by gerouvi**
saas styles refactor

### CODE REVIEW

File: src/components/Auth/SignUp.tsx [hunk 1]
```diff
@@ -97,36 +97,30 @@ const SignUp = () => {
...
+          <Button isLoading={isPending} type='submit' fontSize='sm' variant='brand' fontWeight='500' w='100%' h={50}>
             {t('signup_create_account')}
           </Button>
-        </Box>
+        </Flex>
       </FormProvider>
 
-      <Flex flexDirection='column' justifyContent='center' alignItems='start' maxW='100%' mt='0px'>
-        <Text color={textColorSecondary} fontWeight='400' fontSize='14px'>
+      <Flex flexDirection='column' justifyContent='center' alignItems='start' maxW='100%' mt={0}>
```
Comment (selankon) [inline src/components/Auth/SignUp.tsx:107]: I don't believe that this is necessary ` mt={0}`

File: src/components/OrganizationSaas/Dashboard/AuthBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        minW='100%'
+        maxW={{ base: '90%', md: 96 }}
+        py={5}
+        textAlign='center'
+      >
+        {t('rights')}
+      </Text>
+    </Box>
+  )
+  return (
```
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/AuthBanner.tsx:70]: It never arrives to this return (I think is duplicated from the above return?)

File: src/components/OrganizationSaas/Dashboard/AuthBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+import { ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link as ReactRouterLink } from 'react-router-dom'
+import DarkModeToggle from '~src/themes/saas/components/DarkMode'
+
+interface AuthBannerProps extends ChakraProps {
+  children: ReactNode
+}
+
+const AuthBanner = ({ children, ...props }: AuthBannerProps) => {
```
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/AuthBanner.tsx:11]: Is this a new component? Maybe you could add `defaultValue`  on translations?

File: src/components/OrganizationSaas/Dashboard/Create.tsx [hunk 4]
```diff
@@ -131,48 +108,19 @@ const CreateOrganization = () => (
               </Flex>
             </Grid>
           </Box>
-          <Flex flexDirection='column' alignItems={{ base: 'center', lg: 'start' }} mb='100px' w='full'>
-            <Text fontSize={{ base: '28px', md: '48px', lg: '32px', xl: '48px' }} color='white'>
+          <Flex flexDirection='column' alignItems='center' mb='100px' w='full'>
+            <Text fontSize='5xl' color='white'>
```
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Create.tsx:112]: Should font size to be responsive as on the original?

File: src/components/OrganizationSaas/Dashboard/ProcessesList.tsx [hunk 5]
```diff
@@ -23,22 +23,32 @@ const ProcessesList = ({ loading, processes, error, limit }: ProcessesListProps)
...
+        <Text flexGrow={1} flexShrink={0} flexBasis={36}>
+          TYPE
+        </Text>
+        <Text flexGrow={1} flexShrink={0} flexBasis={24}>
+          STATUS
+        </Text>
+        <Text flexGrow={1} flexShrink={0} flexBasis={24}>
+          VOTERS
+        </Text>
+        <Box flexGrow={1} flexShrink={0} flexBasis={5}></Box>
```
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/ProcessesList.tsx:51]: should be on translations this??

File: src/elements/LayoutAuth.tsx [hunk 6]
```diff
@@ -1,101 +1,63 @@
...
+        </Box>
+      </Flex>
+      <AuthBanner>
+        <Flex flexGrow={1} alignItems='end' justifyContent='center'>
+          <Box mb={24}>
+            <Text fontSize='5xl' color='white'>
+              {t('auth.title')}
+            </Text>
+            <Text fontSize='lg' color='white'>
+              {t('auth.subtitle')}
```
Comment (selankon) [inline src/elements/LayoutAuth.tsx:56]: defaultValues missing

File: src/themes/saas/space.tsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
+export const spacing = {
```
Comment (selankon) [inline src/themes/saas/space.tsx:1]: sorry, this is needed to be specified? I don't know if could works with chakra defaults?

**Review by selankon — CHANGES_REQUESTED** [review]
Hey good job, could you explain a little bit more on the PR description what this PR does? Some brief phrases in order to read fast and understand on which direction this changes goes, if there are new components etc...

Also there are lots of translations without `defaultValue`, we can do it on the future.

File: src/themes/saas/space.tsx (hunk 7 above)
Comment (selankon) [reply src/themes/saas/space.tsx:1]: After a talk with Gerard i knew why is used

**Review by selankon — APPROVED** [review]
LGTM lets see what @jpaulet thinks :)

File: src/components/Auth/SignIn.tsx [hunk 8]
```diff
@@ -79,43 +79,45 @@ const SignIn = () => {
...
       </Flex>
-      <Box pt={2}>
-        <FormControl isInvalid={isError}>
-          {isError && <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>}
-        </FormControl>
-      </Box>
+      {isError && (
+        <Box>
+          <FormControl isInvalid={isError}>
+            <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>
```
Comment (selankon) [inline src/components/Auth/SignIn.tsx:110]: I think we should add the default error message into translations and use it everywhere

File: src/components/Auth/SignUp.tsx [hunk 9]
```diff
@@ -97,36 +97,30 @@ const SignUp = () => {
...
       </Flex>
-      <Box pt={2}>
-        <FormControl isInvalid={isError}>
-          {isError && <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>}
-        </FormControl>
-      </Box>
+      {isError && (
+        <Box>
+          <FormControl isInvalid={isError}>
+            <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>
```
Comment (selankon) [inline src/components/Auth/SignUp.tsx:120]: Implement default error message translation

**Review by selankon — CHANGES_REQUESTED** [review]

**Review by jpaulet — APPROVED** [review]

**Review by selankon — APPROVED** [review]

**Pull request merged**